app/src/main/java/ar/scacchipa/jsonplaceholderviewer/data/ITypicodeSourceData.kt

```diff
@@ -0,0 +1,27 @@
+package ar.scacchipa.jsonplaceholderviewer.data
+
+import com.google.gson.GsonBuilder
+import retrofit2.Response
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+import retrofit2.http.GET
+import retrofit2.http.Path
+
+interface ITypicodeSourceData {
```

Comment:
Maybe there is a better name for this class. Also remember that data sources names should indicate the nature of the datasource, e.g local or remote